dask/tests/test_config.py

```diff
@@ -128,10 +131,34 @@ def test_collect_yaml_paths():
             with open(fn2, "w") as f:
                 yaml.dump(b, f)
 
+            configs = collect_yaml(paths=[fn1, fn2], return_paths=return_paths)
+            if return_paths:
+                assert configs[0] == (pathlib.Path(fn1), a)
+                assert configs[1] == (pathlib.Path(fn2), b)
+            else:
+                assert configs[0] == a
+                assert configs[1] == b
+
```

Comment:
I don't see a reason to parametrize the test? You're already testing with return_paths=False below

Reply:
Indeed, that's true. https://github.com/dask/dask/pull/10930/commits/def44add3a938814050e466ceb2f9aaa75808418